release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3126,6 +3126,19 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.12.1 --pullspecs
 ----
 
+[id="ocp-4-12-1-assisted-installer-api-mixed-arch-clusters"]
+==== Create heterogeneous clusters by using the Assisted Installer API
+
+{product-title} now supports mixed-architecture clusters, also known as heterogeneous clusters, that feature hosts with both AMD64 and AArch64 CPU architectures. Use the {ai-full} API to create a heterogeneous cluster by adding AArch64 worker nodes to a deployed AMD64 cluster. 
```

Comment:
in the spirit of the comment below about ``supports`` term
```suggestion
{product-title} now allows mixed-architecture clusters, also known as heterogeneous clusters, that feature hosts with both AMD64 and AArch64 CPU architectures. Use the {ai-full} API to create a heterogeneous cluster by adding AArch64 worker nodes to a deployed AMD64 cluster. 
```

Reply:
Just reorganised the content to reflect your comment (we try to avoid the word allows)